modules/dxp/apps/commerce-avalara/commerce-avalara-tax-engine-fixed-web/bnd.bnd

```diff
@@ -0,0 +1,4 @@
+Bundle-Name: Liferay Commerce Avalara Tax Engine Fixed Web
+Bundle-SymbolicName: com.liferay.commerce.avalara.tax.engine.fixed.web
+Bundle-Version: 1.0.0
+Web-ContextPath: /commerce-avalara-tax-engine-fixed-web
```

Comment:
Here also
`Liferay-Enterprise-App: dxp.only=true;product.id=9a473157-06a6-44b6-b017-a360ffaf5f38`

Reply:
Got it, will update and fix. Thanks Danny.